openshift/odo
# resolved resource limit not reflecting on unsetting

## What is the purpose of this change? What does it change?
resolved resource limit not reflecting on unsetting.

## Was the change discussed in an issue?
fixes https://github.com/openshift/odo/issues/1576


## How to test changes?

```
odo create java --port 8080 --port 8088
odo config set memory 2Gi
odo push
oc get pod <pod-name> -o yaml # should have memory
odo config unset memory
oc get pod <pod-name> -o yaml # should not have memory
```
